datasets/datasets.json

```diff
@@ -317,6 +317,13 @@
     "type": "jsonl",
     "path": "random-100/"
   },
+  {
```

Comment:
please add also the configuration for experiments with euclidian distance see also random-100-euclidian